scripts/core/ui/components/SearchBar/index.tsx

```diff
@@ -112,26 +112,25 @@ export default class SearchBar extends React.Component<IProps, IState> {
                             <>
                                 {removeButton}
 
-                                <IconButton
-                                    style="outline"
-                                    size="small"
-                                    ariaValue={gettext('Search')}
-                                    icon="chevron-right-thin"
+                                <button
+                                    className="search-start visible"
                                     onClick={() => this.props.onSearch(this.state.searchInputValue)}
-                                />
+                                    aria-label="Start search"
+                                >
+                                    <i className="icon-chevron-right-thin" />
+                                </button>
                             </>
                         )
                         : null
                 )
         );
 
         return (
-            <div className={'SearchBar flat-searchbar' + (searchBarExtended ? ' extended' : '')}>
-                <div className="search-handler" style={{alignItems: 'center'}}>
+            <div className="SearchBar flat-searchbar extended">
+                <div className="search-handler">
                     <label
                         htmlFor={_uniqueId}
                         className="trigger-icon"
-                        onClick={this.toggleSearchBar}
```

Comment:
if change removes usages of a variable, variable declaration has to be removed too